BlazarUI/app/scripts/components/dashboard/StarredBranchesTableRow.jsx

```diff
@@ -9,6 +10,30 @@ import CommitMessage from '../shared/CommitMessage.jsx';
 
 class StarredBranchesTableRow extends Component {
 
+  constructor(props, context) {
+    super(props, context);
+  }
+
+  getRowClassNames(state) {
+    return classNames([
+      tableRowBuildState(state),
+      'clickable-table-row'
+    ]);
+  }
+
+  onTableClick(blazarBranchPath, blazarPath, e) {
+    if (e.target.className === 'sha-link') {
```

Comment:
it's a little more robust to pass this className value in as another argument. In general it's best to keep class names strictly for style, and not mix them up with the logic. Helps minimize potential issues in the future. 

You can also use underscores [partial](http://underscorejs.org/#partial) in these types of cases rather than bind if the context isn't necessary.